packages/react/modules/components/feature-switched/feature-switched.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const isEmptyChildren = children => React.Children.count(children) === 0;
+
+export default class FeatureToggled extends React.PureComponent {
```

Comment:
Wrong name for class?

I find `<FeatureSwitched>` a really weird name, because it is some grammar form that is not present tense or something "Look dude it is a feature but switched", why not:

* `<FeatureSwitch>` - elicits the image of a physical thing
* `<SwitchFeature>` - active imperative voice, nice and clear

Reply:
I went for past tense as `FeatureToggled` is. `SwitchFeature` seems also nice.

Reply:
I kind of find `FeatureToggled` a weird name too 😃, but not weird enough for breaking changes and totally a personal preference. So don't mind me.

Reply:
I agree with @luhmann, `SwitchFeature` is way better. And for `FeatureToggled`, `ToggleFeature` would have been nice. But anyway.. :p

Reply:
And while we're on it, `Variant` instead of `Variate` 🌺

Reply:
You can't believe how happy I am for you all to engage on this. 

1. `FeatureToggled` is a strange name @luhmann @Kerumen 
   - Alright, lets just re-export a `ToggleFeature` and deprecate the other version
   - We can keep `FeatureToggled` for one or more major versions
2. Variant over Variate @dferber90 
   - I am just trying to follow LaunchDarkly's naming here as it was the first adapter and I feel it's the term used in the industry
   - http://blog.launchdarkly.com/tag/multivariate-feature-flags/

Reply:
And what about `variation`?

Reply:
Yeah, we can rename it if we want. I'll do that then. `variate` truly has caused confusion.